include/ofi_shm_p2p.h

```diff
@@ -88,7 +92,11 @@ static struct ofi_shm_p2p_ops p2p_ops[] = {
 		.initialized = false,
 		.init = ofi_shm_p2p_no_init,
 		.cleanup = ofi_shm_p2p_no_cleanup,
+#ifdef _WIN32
 		.copy = ofi_shm_p2p_no_copy,
+#else
+		.copy = cma_copy,
+#endif
```

Comment:
Let's push the _WIN32 checks down.  ofi_cma.h should be includable in any OS.  There's barely any code there.  We could even argue whether cma_copy() could go into ofi_mem.h or some other existing header file.  pid_t is already defined in windows/osd.h.  I think we just need to define the ofi_process_vm_read/writev calls.  Those are defined in the freebsd and osx osd.h files, just not windows.

Reply:
I defined ofi_process_vm_read/writev in indows/osd.h. Compiling it now to see if there are other changes needed.
My preference is to keep ofi_cma.h as it's more explicit and somewhat symmetrical with ofi_xpmem.h.